webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -179,7 +179,7 @@ function isValidFieldForFilter(field: JiraField): boolean {
     }
 
     return allowedTypes.includes(type) || (custom && acceptedCustomTypesForFilters.includes(custom)) ||
```

Comment:
I find this part a bit difficult to follow and understand, but this would be a future refactor.

Reply:
@javaguirre Yeah I too think `option` and `user` should be added to `allowedTypes`. I don't see any logic that would bring if we did that